std/uni.d

```diff
@@ -714,6 +714,8 @@ import std.range.primitives; // back, ElementEncodingType, ElementType, empty,
 import std.traits; // isConvertibleToString, isIntegral, isSomeChar,
     // isSomeString, Unqual
 import std.exception : enforce, collectException;
+import core.memory : pureMalloc, pureRealloc, pureFree;
```

Comment:
Can you use function local imports instead? There are still some issues with top level selective imports: https://issues.dlang.org/show_bug.cgi?id=17630

Reply:
Honestly that was an attempt to solve linker error while using function local imports...